README.md

```diff
@@ -1,472 +1,302 @@
+<img src="https://user-images.githubusercontent.com/15983736/75925917-75d5b580-5e61-11ea-83c8-2a241038e08a.png" align="right" width="150" />
 
-Cache your git credentials by running:
-
-```bash
-git config --global credential.helper osxkeychain
-git config --global user.name <yourname>
-git config --global user.email <your@email.com>
-```
-
-You should also globally ignore your `.DS_Store`, this can be done with:
-
-```bash
-echo ".DS_Store" > ~/.gitignore_global
-git config --global core.excludesfile ~/.gitignore_global
-```
-
-By default `git` sends it's gb output to the system pager, I always would like this disabled for the `git branch` command.
-
-This can be configured (see [here](https://stackoverflow.com/a/48370253)) by running the following command:
-
-```bash
-git config --global pager.branch false
-```
-
-For `git diff` I would like it to output to the system pager if the diff doesn't fit on the terminal screen and output to stdout otherwise.
-
-This can be configured (see [here](https://stackoverflow.com/a/31399632/4699289)) by running:
-
-```bash
-git config --global core.pager "less -FRSX"
-```
-
-----
+Docker desktop can be downloaded from here: https://docs.docker.com/docker-for-mac/install/
 
-#### AWS
+You may have to log in to [`DockerHub`](https://hub.docker.com/) to perform the download.
 
-```bash
-brew install awscli
-# configure your region, access key id and secret access key
-aws configure
-```
+<br><br><br>
 
 ----
 
-#### Node
-
-I install `node` with `brew` - you can use `nvm`, but I don't find I need to change my node version often enough.
-
-Install with:
-
-```bash
-brew install node
-```
+### [Homebrew](https://brew.sh/)
 
-----
+<img src="https://user-images.githubusercontent.com/15983736/75926048-b3d2d980-5e61-11ea-9dbd-d9802a6ba132.png" align="right" width="150" />
 
-#### Elixir
+Follow instructions found here: https://brew.sh
 
-As simple as:
+This is what I ran:
 
 ```bash
-brew install elixir
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
-I've also installed [Phoenix](http://www.phoenixframework.org/) along side this
-
-For the most up to date installation instructions see [here](http://www.phoenixframework.org/docs/installation#section-phoenix)
-
-It can be installed with:
+These are some of the packages I'll always add:
 
 ```bash
-mix archive.install hex phx_new 1.4.0
+brew install git grip fd macvim node python
 ```
 
 ----
 
-#### Python
-
-Although mac comes with `python`, I would recommend upgrading to python3, also we don't have `pip` and other python related tools
-
-Get these by running:
+### [Zsh](https://ohmyz.sh/)
 
-```bash
-brew install python3
-pip3 install --upgrade pip setuptools wheel
-```
+<img src="https://user-images.githubusercontent.com/15983736/75923752-a3206480-5e5d-11ea-9460-e16190f6333c.jpg" alt="oh_my_zsh" align="right" width="150" />
 
-----
+The modern shell supported by an [awesome community](http://ohmyz.sh/).
```

Comment:
`zsh` is now the default shell on Mac: https://support.apple.com/en-us/HT208050 💭 
https://www.theverge.com/2019/6/4/18651872/apple-macos-catalina-zsh-bash-shell-replacement-features

Reply:
Wow, didn't know that!